docs/fleet/provision.md

```diff
@@ -1,278 +1,186 @@
+     - Paste the `viam-server` app config when prompted.
 
-## Provision a new machine
+1. The machine will then disable the hotspot network and attempt to connected using the provided network information.
+   If `viam-agent` cannot establish a connection using the provided network information, the machine will create the hotspot again and continue going through steps (2-5) until a connection is successfully established.
+1. If the connection is successful, `viam-agent` installs `viam-server`.
 
-With the Viam Agent installed, your machine will either connect to a local WiFi network or will create its own WiFi hotspot, depending on your configuration.
+   - `agent-provisioning` will use the provided network if it can connect, even if that network does not have internet access.
+     Note that any features that require internet access will not function if the connected WiFi network is not connected to the internet.
+     If you want `agent-provisioning` to require that a WiFi network be connected to the internet in order to connect to it, enable roaming mode **and** do not provide a primary network.
 
-- If you include a `viam-server` configuration file on your machine, located at <file>/etc/viam.json</file>, which includes a WiFi network and password to connect to, the Viam Agent will connect to the network automatically when in range.
-- If you did not include this file, or the configured WiFi network is not available when your machine comes online, the Viam Agent will create its own WiFi hotspot and request WiFi credentials:
+1. `viam-agent` then starts `viam-server` with a provided configuration and the machine comes **live**.
+   {{% /tab %}}
 
-  ![Diagram of the flow of provisioning a new machine with the Viam agent.](/platform/provision/style-provisioning-diagram.png)
+{{% tab name="With pre-configured network (Roaming mode)" %}}
 
-  If there is an issue connecting to the WiFi hotspot with the input WiFi credentials, this flow will repeat until the device is online:
+Roaming mode allows you to specify multiple WiFi networks **in the machine's or fragment's [`viam-agent` configuration](/configure/agent/#configuration)**, each with their own priority, and `agent-provisioning` will try to connect to each specified network in order of priority from highest to lowest.
 
-  ![Diagram of the flow of provisioning a new machine with the Viam agent when there is an issue connecting to WiFi.](/platform/provision/style-provisioning-diagram-error.png)
+You must enable `roaming_mode` in the [`agent-provisioning` configuration](#configuration) to use roaming mode.
+Additionally, the end user must either **not** provide a primary network or there must be an issue with the connection to the primary network, for `agent-provisioning` to enter roaming mode.
+`agent-provisioning` will enter roaming mode only when `roaming_mode` is enabled and no connection can be established to a primary network.
```

Comment:
As noted above, not true. Roaming mode is enabled when "roaming_mode" is set to true. So this probably doesn't need to be clarified.